ert_gui/gert_main.py

```diff
@@ -38,8 +38,7 @@
 from ert_shared import ERT
 
 from res.enkf import EnKFMain, ResConfig
-
-import ecl
+from ert_shared.services import Storage
 
 
 def run_gui(args):
```

Comment:
is it only when running the GUI you are starting the api now? I guess running the cli should also start off the api?

Reply:
I think the CLI only displays progress information right ? which is not currently part of the database and the API so the question is do we really need to start the API also for the CLI or I am missing something :man_shrugging: ?

Reply:
you can start ert just by running the cli right? so - should that be of any difference from running it with gui? If forward models / workflows are running csv-export - we need the api to be running either on top of ert-storage or dark-storage right? Or am i missing something here?

Reply:
Good point, it might be useful to start up the API also for the CLI in the future.